BookPlayer/Library/LibraryViewController.swift

```diff
@@ -219,7 +235,14 @@ extension LibraryViewController {
         let item = self.items[indexPath.row]
 
         // "…" on a button indicates a follow up dialog instead of an immmediate action in macOS and iOS
-        let deleteAction = UITableViewRowAction(style: .default, title: "Delete…") { (_, indexPath) in
+        var title = "Delete…"
+
+        // Remove the dots if trying to delete an empty playlist
+        if item is Playlist {
```

Comment:
mmmm might as well unwrap it here and avoid 'forcing it' inside
```
if let playlist = item as? Playlist {
  title = (playlist.books?.count ?? 0) > 0 ? title: "Delete"
}
```
To be honest I'd like to replace `.count ?? 0) > 0 ` with a simple `!isEmpty` but that is just for an array not an ordered set 🤔